setup.py

```diff
@@ -7,66 +7,14 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-requirements = [
-    'attrdict',
-    'aws-wsgi',
-    'boto3',
-    'botocore',
-    'certifi',
-    'chardet',
-    'colorama',
-    'connexion',
-    'connexion[swagger-ui]',
-    'docutils',
-    'Flask',
-    'Flask-Cors',
-    'idna',
-    'inflection',
-    'itsdangerous',
-    'Jinja2',
-    'jmespath',
-    'jsonschema==2.5.1',
-    'MarkupSafe',
-    'newrelic',
-    'openapi-spec-validator==0.2.7',
-    'pathlib',
-    'psutil',
-    'pynamodb',
-    'python-dateutil',
-    'python-decouple',
-    'python-json-logger',
-    'PyYAML==5.1.1',
-    'requests',
-    's3transfer',
-    'six',
-    'stripe',
-    'structlog',
-    'urllib3',
-]
+with open('app_requirements.txt') as f:
+    app_requirements = f.read().splitlines()
 
-setup_requirements = [
-    'pytest-runner',
-    'setuptools>=40.5.0',
-]
+with open('setup_requirements.txt') as f:
+    setup_requirements = f.read().splitlines()
 
-test_requirements = [
-    'mockito',
-    'pytest',
-    'pytest-cov',
-    'pytest-mock',
-    'flake8',
-    'mock',
-    'mockito',
-    'mockito',
-    'patch',
-    'pytest',
-    'pytest',
-    'pytest-cov',
-    'pytest-cov',
```

Comment:
Notice that we had multiple, duplicates within this file.  This change set when migrated generated an packaging error on `doit package` that would prevent this moving forward.